discovery/gossiper.go

```diff
@@ -3489,6 +3489,30 @@ func (d *AuthenticatedGossiper) handleAnnSig(ctx context.Context,
 			peerID := nMsg.source.SerializeCompressed()
 			log.Debugf("Got AnnounceSignatures for channel with " +
 				"full proof.")
+			syncer, ok := d.syncMgr.GossipSyncer(
```

Comment:
note that there is a case above where we also send the announcement to the peer (if it is from our local subsystem). That counts as a send for this connection & so we should mark that we've sent it there too